netjsonconfig/backends/raspbian/templates/commands.jinja2

```diff
@@ -0,0 +1,11 @@
+{% for i, j  in data.items() %}
+    {% if i|string() == 'general' %}
+        {% for general in j %}
+            {% if general.get('timezone') %}
+                run commands:
```

Comment:
what about a script? We can then create an install script that must be launched to deploy the configuration